gradle/libs.versions.toml

```diff
@@ -1,89 +1,90 @@
 [versions]
-assetModel = "0.1.2"
-assetSpec = "1.0.0"
-bouncyCastle = "1.70"
+asset-model = "0.1.2"
+asset-specs = "1.0.0"
+blockapi = "0.1.3"
+bouncycastle = "1.70"
 feign = "11.8"
+figure-eventstream = "0.7.1"
```

Comment:
Real change: Bumped to the new eventstream libs from figure.tech versus provenance.io